CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+# Contributing Guide
+
+This repository is a collection of best practices and coding standards. It has been made by [research
+software engineers] at [UCL's Centre for Advanced Research Computing][UCL ARC]. Some of it is from our internal
+documentation and recommendations pulled out and made public. The content is stored in [GitHub flavoured markdown], and
+is rendered as a live [webpage].
+
+Any changes should be reviewed by a member or associate member of ARC. If the additions are taken from ARC's internal
+documentation, **please add the original author** as a reviewer to check that they're OK with their content being made
+public under a CC-by-4.0 license. If something seems like it's broken, please go ahead and [open an issue]!
+
+<!-- prettier-ignore -->
+> [!NOTE]
```

Comment:
This will mean only the `[!NOTE]` is not formatted. The rest will be (which is the bit that isn't broken).